tools/gfx-unit-test/graphics-smoke.slang

```diff
@@ -3,7 +3,8 @@
 // Per-vertex attributes to be assembled from bound vertex buffers.
 struct AssembledVertex
 {
-    float3	position : POSITION;
+    float3	positionA : POSITIONA;
```

Comment:
Instead of changing this shader, which is also used by another test, can you create a new shader file to keep things clean?

Reply:
This shader is only being used by this test as far as I'm aware. This test was still missing actual instancing when I opened the PR for the draw calls.